todo_app/app.py

```diff
@@ -1,15 +1,28 @@
-from flask import Flask
+from flask import Flask, render_template, request # used for jinja and getting post requests 
+
+from todo_app.flask_config import Config 
+
+from todo_app.data.session_items import * # can use * as a wildcard to take everything from another file
+
 
-from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+#display index.html template with list of items via get_items function
+@app.route('/', methods=['GET'])
 def index():
-    return 'Hello World!'
+    items_variable = get_items()
+    return render_template('index.html', items=items_variable)
 
+#Capture form post from /newitem form action in index.html
+@app.route('/newitem', methods=['POST'])
+def addNewItem():
+    newItem = request.form['createNew']
+    add_item(title = newItem)
+    items_variable = get_items()
+    return render_template('index.html', items=items_variable)
```

Comment:
You can replace these last two lines with just `return redirect('/')` (You just need to add "redirect" to your imports from flask)
Which will return a redirect HTTP response telling the browser to load the index page instead.

The benefits:
- Reduce duplicated code
- The user will not see the /newItem url
- The user can refresh the page without resubmitting the form